README.md

```diff
@@ -2,33 +2,40 @@
 
 Welcome!
 
-![OpenXR-Unity-MixedReality-Samples-MainMenu-Cropped](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu-Cropped.jpg)
+![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)
```

Comment:
It looks like this new image is a lower resolution - do you have a higher-resolution image available? The text in the buttons is difficult to read now, and I think it used to function as a nice visual overview of what scenes the BasicSample provides.
https://github.com/microsoft/OpenXR-Unity-MixedReality-Samples/blob/yl/update_readme/Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg

If you want, I could also re-take this photo in the same resolution the image on main is.

Reply:
fixed.